_posts/2023-05-17-otp-26-highlights.md

```diff
@@ -205,27 +204,28 @@ I got the following order:
 ```
 
 There is a new modifier `k` for format strings to specify that maps should
-be sorted before printing:
+be sorted according to the term order of their keys before printing:
 
 ```erlang
 3> io:format("~kp\n", [AM]).
 #{a => 1,b => 2,c => 3}
 ok
 ```
 
-It is also possible to use a custom ordering fun. For example, to
-order the map elements in reverse order based on their keys:
+It is also possible to use a [custom ordering
+fun](https://www.erlang.org/doc/man/io.html#format-1).  For example,
+to order the map elements in reverse order based on their keys:
 
 ```erlang
-4> io:format("~Kp\n", [fun(A, B) -> A > B end,  M]).
+4> io:format("~Kp\n", [fun(A, B) -> A > B end, M]).
 #{c => 3,b => 2,a => 1}
 ok
 ```
 
 There is also a new
 [maps:iterator/2](https://www.erlang.org/doc/man/maps.html#iterator-2)
 function that supports iterating over the elements of the map in a more
-inutive order. Examples will be shown in the next section.
+intutive order. Examples will be shown in the next section.
```

Comment:
Sorry for this, I gave the wrong correction, neither inutive nor intutive are correct, I should have said "intuitive".

Reply:
Fixed.